### ISSUE TITLE
Conda-forge build not working for version 0.8.0

### ISSUE THREAD

**Issue by davkovacs** [libAtoms/matscipy#205]

**Comment by pastewka** [comment]
We don't actively use or support neither `conda` nor `conda-force`. Please talk to the `conda-forge` team if you need `matscipy` there.

@jameskermode - Is this correct or is anyone from your group using `conda`? My past experience with `conda` has been terrible and I generally do not advice using it - my group simply does not use it.

**Comment by jameskermode** [comment]
I completely agree, we also don't use conda at all either. However since the conda-forge recipe exists we should try to keep it up to date (or ask to have it removed). I can try to take a look at this soon.

**Comment by pastewka** [comment]
Okay sounds good. Should we keep this issue open until you have taken a look?

**Comment by jameskermode** [comment]
Let's keep it open for now please

**Comment by jameskermode** [comment]
@davkovacs I think this should now be fixed with the release of v0.8.1, which shows up on conda-forge. If you can confirm this works for you we'll close this issue.
